docs/src/poll.rst

```diff
@@ -85,9 +86,12 @@ API
 .. c:function:: int uv_poll_start(uv_poll_t* handle, int events, uv_poll_cb cb)
 
     Starts polling the file descriptor. `events` is a bitmask consisting made up
-    of UV_READABLE, UV_WRITABLE and UV_DISCONNECT. As soon as an event is detected
-    the callback will be called with `status` set to 0, and the detected events set on the
-    `events` field.
+    of UV_READABLE, UV_WRITABLE, UV_PRIORITIZED and UV_DISCONNECT. As soon as an
+    event is detected the callback will be called with `status` set to 0, and the
+    detected events set on the `events` field.
+
+    The UV_PRIORITIZED event is used to watch for sysfs interrupts or tcp out-of-bound
+    messages.
```

Comment:
A `versionchanged` should also be added. Probably set to `1.11.0`. /cc @saghul ?